lib/modules/manager/bazel-module/extract.ts

```diff
@@ -0,0 +1,14 @@
+import { LooseArray } from '../../../util/schema-utils';
+import type { PackageFileContent } from '../types';
+import { ToBazelDep } from './bazel-dep';
+import { parse } from './parser';
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
+): PackageFileContent | null {
+  const records = parse(content);
+  return LooseArray(ToBazelDep)
+    .transform((deps) => (deps.length ? { deps } : null))
+    .parse(records);
```

Comment:
catch all errors and return null

Reply:
OK. I will add it. However, I have two comments:
1. None of these APIs advertise that they will throw errors. So, adding a `try-catch` without any interesting error handling seems wasteful.
2. I suggest that this type of blanket `try-catch` should reside in the worker. I can understand how the worker should not fail if a manager throws.